### PR TITLE
Clockwork Rebase

### PR DESCRIPTION
**Pull request by Super3222**
This was probably the worst pain I've ever had to go through as a coder...

anyway enjoy this pr. will probably get closed because oisin is handling an entire rebase.

there's probably a little bit more, I have no idea.

Credit goes to
@Xhuis 
@ChangelingRain 
and other /tg/ developers who worked on all of those changes & fixed those bugs after we rebased.

cool pictures I took when testing:

### CODE REVIEW

**Comment by GrayRachnid** [comment]
ur crazy

**Comment by X-TheDark** [comment]
Hot damn, gg.

If this is to be merged, I suggest not squashing this, as it'll make any future rebases (based on Git "rebase" option) easier, as you won't need to hunt the commits that were already merged.

File: code/game/gamemodes/clock_cult/clock_items.dm [hunk 1]
```diff
@@ -832,19 +901,80 @@
...
+		user << "<span class='warning'>[H] has no brain, and thus no mind to claim!</span>"
+		return
+	if(!H.key)
+		user << "<span class='warning'>[H] has no mind to claim!</span>"
+		return
+	playsound(H, 'sound/misc/splort.ogg', 60, 1, -1)
+	playsound(H, 'sound/magic/clockwork/anima_fragment_attack.ogg', 40, 1, -1)
+	H.status_flags |= FAKEDEATH //we want to make sure they don't deathgasp and maybe possibly explode
+	H.death()
+	H.status_flags &= ~FAKEDEATH
```
Comment (X-TheDark) [inline code/game/gamemodes/clock_cult/clock_items.dm]: This needs to be changed to `H.status_flags -= FAKEDEATH` on account of Cruix changing the status_flags from bitflag field to list.

File: code/game/gamemodes/clock_cult/clock_mobs.dm [hunk 2]
```diff
@@ -119,171 +165,259 @@
...
+		combattimer = world.time + initial(combattimer)
+		for(var/mob/living/L in view(2, src))
+			if(istype(L.l_hand, /obj/item/weapon/nullrod) || istype(L.r_hand, /obj/item/weapon/nullrod)) //hand-held holy weapons increase the damage it takes
+				src << "<span class='userdanger'>The presence of a brandished holy artifact weakens your armor!</span>"
+				amount *= 4 //if a wielded null rod is nearby, it takes four times the health damage
+				break
+	return ..() + fatiguedamage
+
+/mob/living/simple_animal/hostile/clockwork/marauder/proc/adjust_fatigue(amount) //Adds or removes the given amount of fatigue
+	if(status_flags & GODMODE)
```
Comment (X-TheDark) [inline code/game/gamemodes/clock_cult/clock_mobs.dm]: `if(GODMODE in status_flags)`

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 3]
```diff
@@ -1030,32 +1234,29 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
...
-	for(var/i in 1 to 10)
-		sleep(10)
+	spawn(total_duration)
 		for(var/mob/living/L in affected_servants)
-			L.maxHealth -= 50
-			L.health -= 50
-			if(L.maxHealth == initial(L.maxHealth))
-				L.stun_absorption = FALSE
+			L << "<span class='notice'>You feel Inath-Neq's power fade from your body.</span>"
+			L.status_flags &= ~GODMODE
```
Comment (X-TheDark) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: `L.status_flags -= GODMODE`

**Comment by ShadowDeath6** [comment]
as long as this removes converting drones

also make a fuckin changelog you weeb

File: config/tips.txt [hunk 4]
```diff
@@ -155,7 +155,32 @@ As a Cultist, do not cause too much chaos before your objective is completed. If
 As a Cultist, your team starts off very weak, but if necessary can quickly convert everything they have into raw power. Make sure you have the numbers and equipment to support going loud, or the cult will fall flat on its face.
 As a Cultist, the Blood Boil rune instantly crits all non-cultists who can see it, but has a chance to explode runes near it, incuding itself.
 As a Cultist, you can create an army of manifested goons using a combination of the Manifest rune, which creates homunculi from ghosts, and the Blood Drain rune, which drains life from anyone standing on any blood drain rune.
-You can deconvert Cultists by feeding them large amounts of holy water.
```
Comment (ShadowDeath6) [inline config/tips.txt:158]: why was this removed

**Comment by ShadowDeath6** [comment]
> no changelog
> not fixing tips
> merged

why live

**Comment by Super3222** [comment]
> why live

i don't know

**Comment by X-TheDark** [comment]
It's better to have this live than not having this live.

**Comment by optimumtact** [comment]
sometimes to make a tasty omelette you have to break the reality of the world you're cooking in.
